sharedse/src/commonJvmMain/kotlin/com/ustadmobile/port/sharedse/contentformats/ContentTypeUtil.kt

```diff
@@ -66,9 +79,31 @@ suspend fun importContainerFromZippedFile(contentEntryUid: Long, mimeType: Strin
     return container
 }
 
+fun createContainerFromDirectory(directory: File, filemap: MutableMap<File, String>): Map<File, String> {
+    val sourceDirPath = Paths.get(directory.toURI())
+    try {
+        Files.walk(sourceDirPath).filter { path -> !Files.isDirectory(path) }
```

Comment:
This won't be available on older versions of Android. We need to stick to java.io.File. 

All we are really supporting is adding one file as the only file of a container here. So we can do this straightforwardly.